Pod/Classes/MWPhotoBrowser.m

```diff
@@ -491,6 +491,16 @@ - (void)viewWillLayoutSubviews {
     }
 }
 
+- (void)viewDidLayoutSubviews {
```

Comment:
ist das nicht suboptimal in `viewDidLayoutSubviews`? dann werden die subviews doch doppelt gelayoutet

Reply:
hmm ja ich habs noch mit `viewWillTransitionToSize` versucht aber da hat das nicht funktioniert weil die neue Size eben noch nicht applied ist.

Reply:
ach vergiss :) 588bd294e633772a915e1610e74df82cff27da52